manager-rs/directory-service/build.rs

```diff
@@ -0,0 +1,7 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::configure()
+        .build_client(true)
```

Comment:
Nit-- you probably only need to build the server as part of the directory service?

Reply:
Fixed